backend/src/db/migrations/31-series-metadata.sql

```diff
@@ -0,0 +1,63 @@
+-- Adds `metadata` and `created` fields to `series`. Also adjusts the
+-- `check_metadata_format` function from `events` to be usable in both
+-- contexts, and adds new triggers.
+
+alter table series
+    add column created timestamp with time zone,
+    add column metadata jsonb;
+
+drop trigger check_metadata_format_on_insert on events;
+drop function check_metadata_format;
+
+create or replace function check_metadata_format() returns trigger as $$
```

Comment:
Could use a comment describing what this function does, roughly. Not all the details. An importantly, mention that one argument is expected and what it's for.